engine/include/systems/physics_system.hpp

```diff
@@ -10,16 +19,21 @@ class PhysicsSystem : public System
     PhysicsSystem(ECS& ecs, PhysicsModule& physicsModule);
     ~PhysicsSystem() = default;
 
-    entt::entity CreatePhysicsEntity();
-    void CreatePhysicsEntity(RigidbodyComponent& rb);
     void AddRigidBody(entt::entity entity, RigidbodyComponent& rigidbody);
+    [[nodiscard]] RayHitInfo ShootRay(const glm::vec3& origin, const glm::vec3& direction, float distance) const;
     void CleanUp();
 
     void Update([[maybe_unused]] ECS& ecs, [[maybe_unused]] float deltaTime) override;
     void Render([[maybe_unused]] const ECS& ecs) const override;
     void Inspect() override;
+    void InspectRigidBody(RigidbodyComponent& rb);
+
+    void SetCameraPosition(const glm::vec3& position) { _cameraPosition = position; }
```

Comment:
what does the camera inside the physics system do?

Reply:
My bad, i forgot it there when i was trying something